src/gui/DatabaseWidget.cpp

```diff
@@ -2021,39 +2024,70 @@ void DatabaseWidget::processAutoOpen()
         if (entry->url().isEmpty() || (entry->password().isEmpty() && entry->username().isEmpty())) {
             continue;
         }
-        QFileInfo filepath;
-        QFileInfo keyfile;
-        QString databaseUrl = entry->resolveMultiplePlaceholders(entry->url());
 
-        if (databaseUrl.startsWith("file://")) {
-            QUrl url(databaseUrl);
-            filepath.setFile(url.toLocalFile());
-        } else {
-            filepath.setFile(databaseUrl);
-            if (filepath.isRelative()) {
-                QFileInfo currentpath(m_db->filePath());
-                filepath.setFile(currentpath.absoluteDir(), databaseUrl);
+        auto ifDevice = entry->attribute("ifDevice");
```

Comment:
This needs documentation and should be implemented for SSHAgent as well.

Reply:
We'll add a section for "Opening Databases within Databases" in the Ascii Docs to cover both AutoOpen and this new feature.